aiohttp/web_response.py

```diff
@@ -259,9 +259,17 @@ def last_modified(
             self._headers.pop(hdrs.LAST_MODIFIED, None)
         elif isinstance(value, (int, float)):
             self._headers[hdrs.LAST_MODIFIED] = time.strftime(
-                "%a, %d %b %Y %H:%M:%S GMT", time.gmtime(math.ceil(value))
+                "%a, %d %b %Y %H:%M:%S GMT", time.gmtime(math.floor(value))
```

Comment:
The floor is unneeded, gmtime() already does that.
```suggestion
                "%a, %d %b %Y %H:%M:%S GMT", time.gmtime(value)
```